### ISSUE TITLE
xdescribe and xcontext does't have default parameter for flags

### ISSUE THREAD

**Issue by ddddxxx** [Quick/Quick#945]
- [x] I have read [CONTRIBUTING](https://github.com/Quick/Quick/blob/master/CONTRIBUTING.md) and have done my best to follow them.

### What did you do?

Prepend "x" on describe("...") {}

### What did you expect to happen?

The describe closure is marked as pending, and all examples within the closure is disabled.

### What actually happened instead?

Compilation error: missing argument for parameter 'flags' in call.

### Environment

List the software versions you're using:

 - Quick: v2.2.0
 - Nimble: v8.0.2
 - Xcode Version: 11.3
 - Swift Version: 5.1
 - Swift Package Manager *5.1.0*

### Project that demonstrates the issue

```swift
class TestSpec: QuickSpec {
    override func spec() {
        xdescribe("...") {}
    }
}
```

---

Source code regarding this issue: https://github.com/Quick/Quick/blob/master/Sources/Quick/DSL/DSL.swift#L201

I'd like to pr. I'm just checking here to make sure I'm not missing something obvious.

**Comment by pobengtsson** [comment]
There is actually a PR for this. https://github.com/Quick/Quick/pull/944

**Comment by ikesyo** [comment]
Addressed by #944.
